Automattic/wp-calypso
# Jetpack: Cobrand connection flow for Liquid Web sites

#### Changes proposed in this Pull Request

* Add Liquid Web partner keys to `getPartnerSlugFromQuery` selector
* Add new `JetpackLiquidWebLogo` component which houses Liquid Web's SVG logo

#### Testing instructions



* Click "Set Up Jetpack" button on disconnected site
* Swap `wordpress.com` in connection URL with `calypso.localhost:3000`. Also, set `&partner_id=60179`. 
* Load that URL
* Ensure that Liquid Web logo id displayed next to Jetpack logo
* Go to http://calypso.localhost:3000/devdocs/design/jetpack-header and ensure the Liquid Web cobranded logo shows properly

It should look something like this:

<img width="464" alt="screen shot 2019-01-10 at 2 04 51 pm" src="https://user-images.githubusercontent.com/1126811/50995140-cf538600-14e3-11e9-9f75-f9f4ae916565.png">
